homework/src/Contributor.js

```diff
@@ -13,6 +13,16 @@ class Contributor {
    * @param {HTMLElement} contributorList The parent element in which to render the contributor.
   */
   render(contributorList) {
-    // Replace this comment with your code
+    const li = Util.createAndAppend('li', contributorList, { 'aria-label': this.data.login, 'tabindex': '0' });
+    li.addEventListener("click", () => { window.open(this.data.html_url); });
+    li.addEventListener("keypress", () => {
```

Comment:
See my comment in week 2 for an alternative using the `<a>` element to make this navigatable with the tab key.